Services/Curl.php

```diff
@@ -95,7 +95,9 @@ public function sendRequest($url, $method, array $options = array(), $payload =
         $curlMetaData = (object) curl_getinfo($this->curl);
 
         $this->curlOptionsHandler->reset();
-        function_exists(curl_reset($this->curl)) ? curl_reset($this->curl) : curl_setopt_array($this->curl, $this->curlOptionsHandler->getOptions());
+        if (function_exists('curl_reset')) {
```

Comment:
Far far away :D
We are located in Koblenz, western Germany. If you want to you can add me on skype.
tobias.teeage-beatz